MentoringHomeWork/HW4/Task1/Calculate/Calculate.cs

```diff
@@ -0,0 +1,31 @@
+namespace Task1.Calculate;
+
+public class Calculate
```

Comment:
You should think in terms of what this class does—validate smth: Validator, help with smth: Helper, model of smth: BookModel, and so on.
It should be something like a Calculator. Please rename.